### PR TITLE
hw-mgmt: scripts: add DPU and Comex AMD to SMBIOS BOM plus improvements.

### PR DESCRIPTION
**Pull request by mshych**
Add BF3 DPU to SMBIOS BOM.
Add AMD Comex to SMBIOS BOM.
Generalization of multiple boards processing.
Code clean.

Signed-off-by: Michael Shych <michaelsh@nvidia.com>

### CODE REVIEW

File: usr/usr/bin/hw-management-devtree.sh [hunk 1]
```diff
@@ -542,8 +583,28 @@ devtr_check_supported_system_init_alternatives()
 			done
 			return 0
 			;;
+		VMOD0019)
+			case $sku in
+				HI160)
```
Comment (vadimp-nvidia) [inline usr/usr/bin/hw-management-devtree.sh:588]: You have redundant tabs here.

File: usr/usr/bin/hw-management-devtree.sh [hunk 2]
```diff
@@ -552,11 +613,10 @@ devtr_check_supported_system_init_alternatives()
...
 	local board_str=$1
-	local clk_board_num=1
-	local clk_bus_offset=0
-	local clk_addr_offset=0
-	local swb_board_num=1
-	local swb_bus_offset=0
+	local board_num=1
+	local board_bus_offset=0
+	local board_name_pfx=
+#	local board_addr_offset=0
```
Comment (vadimp-nvidia) [inline usr/usr/bin/hw-management-devtree.sh:619]: It seems leftover comment for board_addr_offset

Comment (mshych) [reply usr/usr/bin/hw-management-devtree.sh:619]: No, It's not leftover but I specially left it commented as an example.
There is also a comment:
# There is no currently address offset. Leave commented just for possible future use

File: usr/usr/bin/hw-management-devtree.sh (hunk 1 above)
Comment (mshych) [reply usr/usr/bin/hw-management-devtree.sh:588]: Strange, I checked and see it correct both in file itself and in github diff
diff

Comment (mshych) [reply usr/usr/bin/hw-management-devtree.sh:588]: It's the same as in all function, other cases:
diff

**Review by vadimp-nvidia — APPROVED** [review]

**Pull request merged**